mythril/analysis/report.py

```diff
@@ -115,6 +126,8 @@ def add_code_info(self, contract):
             self.filename = codeinfo.filename
             self.code = codeinfo.code
             self.lineno = codeinfo.lineno
+            if self.lineno is None:
```

Comment:
```suggestion
            if codeinfo.solidity_file_idx == -1:
```
Should be equivalent, but I think this is a bit more self-explanatory

Reply:
what do you think?

Reply:
I think codeinfo and codeinfo.filename attribute doesn't have solidity_file_idx.

Reply:
ah I had mistaken the type, what about `codeinfo.solc_mapping.solidity_file_idx`